### PR TITLE
[PID Data Model] PID MDOC-CBOR section

### LINKED ISSUE #18: Pid EAA mdoc-cbor chapter
https://github.com/italia/eidas-it-wallet-docs/blob/0a918c452512f9b77887b2748b75cffb98579558/docs/en/pid-eaa-mdoc-cbor.rst?plain=1#LL1

### PR DESCRIPTION
**Pull request by peppelinux**
this PR resolves https://github.com/italia/eidas-it-wallet-docs/issues/18

## Review

- [x] Ensure your files are written following RST specs (*not MD!*)
- [ ] Italian version
- [x] English version
- [ ] Example files 
- [ ] Ask for review

### CODE REVIEW

File: docs/en/pid-data-model.rst [hunk 1]
```diff
@@ -393,8 +395,295 @@ The combined format for the PID issuance is given by
...
+            "eu.europa.ec.eudiw.pid.it.1": [
+                {
+                    "digestID": 0,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "tax_id_code",
+                    "elementValue": "TINIT-XXXXXXXXXXXXXXXX"
+                },
+            ],
+            ]
+        },
```
Comment (balanza) [inline docs/en/pid-data-model.rst:499]: Should we be consistent on claims regardless of data format? In https://italia.github.io/eidas-it-wallet-docs/en/pid-data-model.html we used `tax_id_number`, `place_of_birth`, etc.

Comment (peppelinux) [reply docs/en/pid-data-model.rst:499]: I fully agree with you but unfortunately the PID rulebook is stucked on the defined attributes and approaches of ISO 18013-5

regarding `tax_id_code` vs `tax_id_number` we found a typo @fmarino-ipzs 
in a section is defined with a name and in other with the other name

Comment (peppelinux) [reply docs/en/pid-data-model.rst:499]: I vote for `code` since is it contains both numbers and letters

**Comment by peppelinux** [comment]
This PR requires the additional claims defined in the PID rulebook, similar to the one we have in SD-JWT as IDA

File: docs/en/pid-data-model.rst (hunk 1 above)
Comment (peppelinux) [reply docs/en/pid-data-model.rst:499]: typo resolved here https://github.com/italia/eudi-wallet-it-docs/pull/64

@fmarino-ipzs @balanza ^

**Comment by peppelinux** [comment]
overseeded by https://github.com/italia/eudi-wallet-it-docs/pull/166

**Pull request closed without merging**